homeassistant/const.py

```diff
@@ -387,9 +387,9 @@
 CURRENCY_CENT = "¢"
 
 # Temperature units
-TEMP_CELSIUS = f"{DEGREE}C"
-TEMP_FAHRENHEIT = f"{DEGREE}F"
-TEMP_KELVIN = f"{DEGREE}K"
+TEMP_CELSIUS = "°C"
+TEMP_FAHRENHEIT = "°F"
+TEMP_KELVIN = "°K"
```

Comment:
According to https://github.com/home-assistant/core/pull/40574 it should be without the “degree” symbol.

Reply:
Thanks for the hint :+1: